CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v31.6.0...main)
 
+* General
+  * Limit ping request body size to 1MB. ([#1098](https://github.com/mozilla/glean/pull/1098))
+* iOS
+  * Implement ping tagging (i.e. the `X-Source-Tags` header) through custom URL ([#1100](https://github.com/mozilla/glean/pull/1100)).
```

Comment:
:D was that missed in that PR? Good to still get it in.

Reply:
Yeap, I am trying to remind myself everytime, but still miss it sometimes.